tester/Dockerfile.plasma_deployer

```diff
@@ -0,0 +1,17 @@
+FROM node:8.16-alpine
+
+MAINTAINER OmiseGO Engineering <eng@omise.co>
+
+WORKDIR /home/node
+
+RUN apk add --update \
+    python \
+    python-dev \
+    py-pip \
+    build-base \
+		git
+
+RUN git clone https://github.com/omisego/plasma-contracts.git
+RUN cd /home/node/plasma-contracts && git reset --hard ea36f5ff46ab72ec5c281fa0a3dffe3bcc83178b
```

Comment:
It turns out the failed test in https://github.com/omisego/elixir-omg/pull/1056 is due to this line of code. This is pulling `plasma-contracts` that's about 10 days old. There's been a quite a bit of changes since then.

Can this be bumped to https://github.com/omisego/plasma-contracts/commit/4fe49008625181df6a9b81f0357f849ccf413acd or even better https://github.com/omisego/plasma-contracts/commit/a6d870f4562b359bece7bea01643e4a6b567b5a9 with the `MIN_EXIT_PERIOD` support?

Reply:
```suggestion
RUN cd /home/node/plasma-contracts && git reset --hard 1608296ef56ab98d857321a687c10dcdd86d72d3
```

@unnawut ? wdyt?

Reply:
Yes lovely please <3